packages/@react-spectrum/slider/docs/Slider.mdx

```diff
@@ -0,0 +1,162 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/slider';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/slider/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Slider} from '@react-spectrum/slider';
+```
+
+---
+category: Forms
+---
+
+# Slider
+
+<p>{docs.exports.Slider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Slider']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/slider/'}]} />
+
+## Example
+
+```tsx example
+<Slider label="Cookies to buy" defaultValue={12} />
+```
+
+## Value
+
+Sliders are controlled with the `value` and `defaultValue` props which must fall between
+the minimum and maximum values which default of 0 and 100, respectively.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState(25);
+  return (
+    <Flex gap="size-150">
+      <Slider
+        label="Cookies to buy (Uncontrolled)"
+        defaultValue={25} />
+      <Slider
+        label="Cookies to buy (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue` props.
+
+```tsx example
+<Slider
+  label="Cookies to buy"
+  minValue={50}
+  maxValue={150}
+  defaultValue={100} />
+```
+
+Values are formatted as plainly by default, but this can be modified by using the `formatOptions` prop to specify a different format.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied based on the current locale.
+
+```tsx example
+<Slider
+  label="Currency"
+  formatOptions={{style: 'currency', currency: 'JPY'}}
+  defaultValue={60} />
+```
+
+### Accessibility
+
+Depending on the visualization being used (i.e. depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` prop is required.
```

Comment:
Wonder if it should be changed here too https://react-spectrum.adobe.com/react-spectrum/Meter.html#accessibility
(I was able to lift a lot of work from Meter)

Reply:
Hmm, IMO it should be changed in Meter as well and expanded so that there is an example of what exactly the aria-label/labelledby should say if `showValueLabel` is false (does the aria-label/labelledby state the title of the Meter/Slider   _and_ the current value?)

Open to discussion, happy for others to weigh in. Doesn't need to be handled in this PR necessarily as well

Reply:
So I'm actually a little unclear on the valueLabel, my VO isn't reading it off at all no matter what happens